### PR TITLE
Bump 0.1.3

### PR DESCRIPTION
**Pull request by joshmoore**
@jburel @will-moore -- how does this Dockerfile look in terms of the standard release process? My thinking would be that the sdist that is created could be `twine upload`-ed.

### CODE REVIEW

**Comment by joshmoore** [comment]
(I don't have permission to upload to https://test.pypi.org/project/omero-parade/)

**Comment by jburel** [comment]
To avoid publishing other version(s) of the app (i tested the Dockerfile without cleaning first), we should add 
``yum install -y git``
and run ``git clean -dfx`` before building
See https://github.com/ome/ome-documentation/blob/develop/contributing/tools-maintenance.rst

**Comment by jburel** [comment]
tested the upload via twine  see https://test.pypi.org/project/omero-parade/

**Comment by jburel** [comment]
> (I don't have permission to upload to https://test.pypi.org/project/omero-parade/)
I have added you as an owner of omero-parade. 
I have not used testpypi for a while

**Comment by jburel** [comment]
Other apps like omero-iviewer now use travis to deploy 
e.g.
```
deploy:
  provider: pypi
  user: $PYPI_USER
  password: $PYPI_PASSWORD
  on:
    tags: true
```

File: .omeroci/app-deps [hunk 1]
```diff
@@ -3,4 +3,4 @@
 set -e
 set -u
 yum -y install https://rpm.nodesource.com/pub_6.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm
-yum -y install npm
+yum -y install nodejs
```
Comment (jburel) [inline .omeroci/app-deps]: This is a new problem. This matches what is needed on iviewer. We may want in the feature to install a newer version of nodejs

**Comment by jburel** [comment]
Docker file is a useful addition

**Comment by jburel** [comment]
If no objections, I will merge this PR and tag the repository

**Comment by joshmoore** [comment]
WFM.

**Pull request merged**